### PR TITLE
Image control

### PR DESCRIPTION
**Pull request by gdt050579**
- Support for image preview (translate to internal color set)
- Add several extra keys for navigation in CanvasControl 
- Add a demo that shows and image from Dizzy (Z80 game)

### CODE REVIEW

File: AppCUI/src/Controls/CMakeLists.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ target_sources(AppCUI PRIVATE
 	Control.cpp 
 	Label.cpp 
 	ListView.cpp
+        ImageViewer.cpp
```
Comment (beaverden) [inline AppCUI/src/Controls/CMakeLists.txt:9]: se pare ca ai un tab un plus aici

File: AppCUI/src/Controls/CanvasViewer.cpp [hunk 2]
```diff
@@ -4,6 +4,25 @@ using namespace AppCUI::Controls;
 using namespace AppCUI::Graphics;
 using namespace AppCUI::Input;
 
+void CanvasControlContext::MoveScrollTo(int newX, int newY)
+{
+    int imgWidth = canvas.GetWidth();
```
Comment (beaverden) [inline AppCUI/src/Controls/CanvasViewer.cpp:9]: Poti sa le pui const dacă tot nu le modifici

Comment (gdt050579) [reply AppCUI/src/Controls/CanvasViewer.cpp]: Fixed

File: AppCUI/src/Controls/CMakeLists.txt (hunk 1 above)
Comment (gdt050579) [reply AppCUI/src/Controls/CMakeLists.txt]: Fixed

**Pull request merged**